imports/components/@primitives/UI/live/index.js

```diff
@@ -113,11 +131,28 @@ const map = ({ live }) => ({ live });
 
 const withRedux = connect(map);
 
+// const withLive = graphql(LIVE_QUERY, {
+//   options: { pollInterval: 60000 },
+// });
+
+// XXX FOR TESTING ONLY
 const withLive = graphql(LIVE_QUERY, {
```

Comment:
Lets comment this out but leave it in the file for testing. Essentially lets only merge working code